### PR TITLE
feat: Added json output for node create

### LINKED ISSUE #4967: Define json output for `ockam node create`
### Current behavior

This does not output json

```
» ockam node create --output json

Node:
  Name: 74096240
  Status: UP
  Route To Node:
    Short: /node/74096240
    Verbose: /dnsaddr/localhost/tcp/53783
...
```

### Desired behavior

Change the code here:

https://github.com/build-trust/ockam/blob/develop/implementations/rust/ockam/ockam_command/src/node/create.rs

to output json describing the node.

We want to start using this in the [typescript wrapper](trust/ockam/blob/ec7268aeef9b59bb22f3ae36a7515bd03ec09a4b/implementations/typescript/ockam/ockam_command/src/node.ts#L26) around command, so as a start json output showing the generated name of the node would be very useful.

Would allow us to make name here optional:

https://github.com/build-trust/ockam/blob/develop/implementations/typescript/ockam/ockam_command/src/node.ts#L26

---

_We love helping new contributors!_
_If you have questions or need help as you work on your first Ockam contribution, please leave a comment on [this discussion](https://github.com/ockam-network/ockam/discussions/1081)._
_If you're looking for other issues to contribute to, checkout [this discussion](https://github.com/build-trust/ockam/discussions/3206) and labels -  https://github.com/build-trust/ockam/labels/good%20first%20issue or https://github.com/build-trust/ockam/labels/help%20wanted_

### PR DESCRIPTION
**Pull request by gautamprikshit1**
## Proposed changes

fixed #4967 

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/src/node/show.rs [hunk 1]
```diff
@@ -72,6 +73,15 @@ fn print_node_info(
 ) {
     println!();
     println!("Node:");
+    if opts.global_args.output_format == OutputFormat::Json {
+        opts.terminal
+            .clone()
+            .stdout()
+            .json(serde_json::json!({"node": { "name": &node_name }}))
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_command/src/node/show.rs]: @CAOakleyII after your recent work at https://github.com/build-trust/ockam/pull/5080, I was wondering if we should flatten the structure here to just `{ "name": &node_name }`

**Review by adrianbenavides — COMMENTED** [review]
Hey @gautamprikshit1 , thanks a lot for submitting this PR!

Replying to your question here https://github.com/build-trust/ockam/issues/4967#issuecomment-1584128818, you can run `cargo run --package ockam_command --bin ockam -- node create --output json` to check your changes.

Additionally, it would be great if you could add a test [here](https://github.com/build-trust/ockam/blob/b4b3f522970adfae8bd324e18a8ade1234d336b2/implementations/rust/ockam/ockam_command/tests/bats/unit.bats) to partially check the output. There are other tests doing similar things that you can use as reference.

**Comment by adrianbenavides** [comment]
@gautamprikshit1 while we wait for @CAOakleyII answer, make sure to add yourself into the contributors list and set up your account to verify your commits:

- [ ] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [ ] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

Let us know if you have any questions!

**Comment by adrianbenavides** [comment]
@gautamprikshit1 just wanted to ping you about this PR. We can merge it as soon as you add yourself to the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file. Let me know if I can help you with anything 👍 

> I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

**Comment by gautamprikshit1** [comment]
@adrianbenavides done

**Review by adrianbenavides — APPROVED** [review]
Awesome, thanks @gautamprikshit1 and congrats on your first merged PR! Adding it to the merge queue 🚂

**Review by polvorin — APPROVED** [review]

**Pull request merged**